pkg/executor/executortype/container/containermgr.go

```diff
@@ -240,7 +240,7 @@ func (caaf *Container) RefreshFuncPods(ctx context.Context, logger *zap.Logger,
 
 	funcLabels := caaf.getDeployLabels(f.ObjectMeta)
 
-	dep, err := caaf.kubernetesClient.AppsV1().Deployments(metav1.NamespaceAll).List(ctx, metav1.ListOptions{
+	dep, err := caaf.kubernetesClient.AppsV1().Deployments(f.Spec.Environment.Namespace).List(ctx, metav1.ListOptions{
```

Comment:
For container functions, I think environment details are empty. Probably we should `f.Namespace` here.

Reply:
done